src/status_im/navigation/screens.cljs

```diff
@@ -320,8 +320,7 @@
    {:name      :screen/settings.keycard
     :metrics   {:track?   :true
                 :alias-id :settings.keycard}
-    :options   {:theme  :dark
-                :insets {:top? true :bottom? true}}
+    :options   options/transparent-modal-screen-options
```

Comment:
this is not needed anymore, same as quo/overlay and insets (safe-area/get-insets)

Reply:
we still don't have requirement, so any version will be ok, so we need to implement the simplest one, which is in develop right now